src/lib/consensus/proof_of_stake.ml

```diff
@@ -343,8 +342,13 @@ module Data = struct
           ; ( "curr_epoch_snapshot"
             , [%to_yojson: Snapshot.t] t.curr_epoch_snapshot )
           ; ( "last_checked_slot_and_epoch"
-            , [%to_yojson: Epoch.t * Epoch.Slot.t]
-                t.last_checked_slot_and_epoch )
+            , `Assoc
```

Comment:
`[%to_yojson (Public_key.Compressed.t * (Epoch.t * Epoch.Slot.t)) list] (Public_key.Compressed.Table.to_alist ...)`?

Reply:
doesn't work, `to_yojson` doesn't like something here. Plus we'd want this to be an object which means we need the assoc.